docs/get-started.md

```diff
@@ -139,23 +146,3 @@ Make sure to specify the machine name as an argument:
 
     $ docker-machine stop dev
     $ docker-machine start dev
-
```

Comment:
Don't worry, this hasn't been removed: It's been moved to the landing page to ensure that it's presented clearly and at the top level.